src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/ImageComponent.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { useSelector } from 'react-redux';
+import { Grid, makeStyles } from '@material-ui/core';
+import { ITextResourceBindings } from 'src/features/form/layout';
+import { HelpTextContainer } from 'src/features/form/components/HelpTextContainer';
+import { IRuntimeState } from '../../types';
+
+export interface IImageProps {
+  id: string;
+  imageSpecs: IImagesSpecifics;
+  textResourceBindings: ITextResourceBindings;
+  commonPhrases: any;
+  getTextResource: (key: string) => string;
+  getTextResourceAsString: (key: string) => string;
+}
+
+export interface IImagesSpecifics {
+  imageSrc: IImageSrc;
+  imgWidth: string;
+  align: string;
+}
+
+export interface IImageSrc {
+  nb: string;
+  nn: string;
+  en: string;
+}
+
+const useStyles = makeStyles({
+  spacing: {
+    letterSpacing: '0.3px',
+  },
+});
+
+export function ImageComponent(props: IImageProps) {
+  const classes = useStyles();
+  const language: string =
+    useSelector((state: IRuntimeState) => state.profile.profile.profileSettingPreference.language);
+  const width = props.imageSpecs.imgWidth || '100%';
+  const altText = props.getTextResourceAsString(props.textResourceBindings.altTextImg) || 'image';
+
+  let imgSrc;
+  if (language === 'en' && props.imageSpecs.imageSrc.en) {
+    imgSrc = props.imageSpecs.imageSrc.en;
+  } else if (language === 'nn' && props.imageSpecs.imageSrc.nn) {
+    imgSrc = props.imageSpecs.imageSrc.nn;
+  } else {
+    imgSrc = props.imageSpecs.imageSrc.nb;
+  }
+
+  let align;
+  if (props.imageSpecs.align === 'right') {
+    align = 'flex-end';
+  } else if (props.imageSpecs.align === 'left') {
+    align = 'flex-start';
+  } else {
+    align = 'center';
+  }
+
+  let showImg;
+  const imgType = imgSrc.slice(-3);
+  if (imgType.toLowerCase() === 'svg') {
+    showImg =
+      <object
+        type='image/svg+xml'
+        id={props.id}
+        data={imgSrc}
+        width={width}
+      >
+        <img
+          src={imgSrc}
+          alt={altText}
+        />
+      </object>;
+  } else {
+    showImg =
+      <img
+        id={props.id}
+        src={imgSrc}
+        alt={altText}
+        width={width}
+      />;
+  }
+  return (
+    <Grid
+      container
+      direction='row'
+      justify={align}
+    >
+      <Grid item={true}>
+        {showImg}
+      </Grid>
+      {props.textResourceBindings?.help &&
+      <Grid item={true} className={classes.spacing}>
```

Comment:
Have you tested this feature? Does this look alright? My fear was that the help icon would be misaligned.

If this however works as expected, sure, well just keep it. Otherwise, I suggest we add this sometime in the future if this need arises.

Reply:
Yeah, I mean it looks about the same as with other components.
Like this:
![image](https://user-images.githubusercontent.com/22012997/131975410-eaaaa444-2ab2-4ddd-86dd-80403e1de545.png)
Whether or not it makes sense to have a help text on a picture is another question. But I figured that would be up to the one making the form.

Reply:
Agreed. Keep it!